packages/server/api/src/app/flows/flow.module.ts

```diff
@@ -45,6 +48,13 @@ export const flowModule: FastifyPluginAsyncTypebox = async (app) => {
         );
         socket.emit(WebsocketClientEvent.TEST_FLOW_RUN_STARTED, flowRun);
       } catch (err) {
+        sendWorkflowTestFailureEvent({
+          userId: principal?.id ?? '',
+          projectId: principal?.projectId ?? '',
+          flowVersionId: data.flowVersionId,
```

Comment:
I think it's worth adding the other params too,
            flowRunId: flowRun.id,
            flowId: flowRun.flowId,